plugins/cake/cake_tin_ecn_

```diff
@@ -0,0 +1,365 @@
+#!/bin/bash
```

Comment:
The plugins `cake_tin_throughput_`, `cake_tin_ecn_`, `cake_tin_delay_` and `cake_tin_pps_` seem to share almost all of their code.
I would really like to avoid this code duplication.
How could you merge them? Maybe by using a multigraph plugin? :)

Reply:
Can I generate multiple level 0 graphs with one multigraph plugin? With different nested level 1 graphs?

Reply:
Yes, you can populate any namespace you like from a multigraph plugin.

Reply:
@niclan thanks, this was very helpful. :)